config/environments/development.rb

```diff
@@ -48,6 +48,12 @@
   # config.action_view.raise_on_missing_translations = true
   ConfigHelper.setup_action_mailer(config)
 
+  # Turns on SQL output in local dev Rails console sessions
+  config.semantic_logger.add_appender(
+    io: STDOUT,
+    level: :debug,
+    formatter: config.rails_semantic_logger.format
+  )
```

Comment:
```
if env['SOME_ENVIRONMENT_FLAG_ONLY_YOU_HAVE_SET_LOCALLY']
     # Turns on SQL output in local dev Rails console sessions
  config.semantic_logger.add_appender(
    io: STDOUT,
    level: :debug,
    formatter: config.rails_semantic_logger.format
  )
end
```

Reply:
If there's no way around the duplication effort, and we desire the ability to see the SQL output in the console, then this is the solution that makes the most sense. We have a variable `SKIP_COVERAGE` which functions similarly - i.e. I'll skip coverage when running specs within `vim` to save time.